src/Uno.UI/UI/Xaml/DragRoot.cs

```diff
@@ -0,0 +1,73 @@
+#nullable enable
+
+using System;
+using System.Linq;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+
+namespace Windows.UI.Xaml
+{
+	internal partial class DragRoot : Canvas
+	{
+		private readonly DragDropManager _manager;
+
+		public DragRoot(DragDropManager manager)
+		{
+			_manager = manager;
+
+			VerticalAlignment = VerticalAlignment.Stretch;
+			HorizontalAlignment = HorizontalAlignment.Stretch;
+			Background = new SolidColorBrush(Colors.Transparent);
+
+			//PointerEntered += OnPointerEntered;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6270104)